code/modules/pda/PDA.dm

```diff
@@ -31,6 +31,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 	var/datum/data/pda/utility/scanmode/scanmode = null
 
 	var/lock_code = "" // Lockcode to unlock uplink
+	var/silent = FALSE //To beep or not to beep, that is the question
```

Comment:
I moved silent to the PDA level because fetching the messenger program every time you need to modify this is ridiculous---also, frankly, this is a case of making PDA's *too* modular; having the messenger silence on the program itself, and only the program means we can't utilize it for other stuff, like this.